Tests/Tests/STPPaymentMethodFunctionalTest.m

```diff
@@ -0,0 +1,92 @@
+//
+//  STPPaymentMethodFunctionalTest.m
+//  StripeiOS Tests
+//
+//  Created by Yuki Tokuhiro on 3/6/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "STPNetworkStubbingTestCase.h"
+
+@import Stripe;
+
+@interface STPPaymentMethodFunctionalTest : STPNetworkStubbingTestCase
+
+@end
+
+@implementation STPPaymentMethodFunctionalTest
+
+- (void)setUp {
+//    self.recordingMode = YES;
```

Comment:
debugging?

Reply:
I've seen this in a few other functional tests and it was helpful for me, so I'm leaning towards keeping it.  I don't know if there's a particular reason this is in some but not all tests.

Reply:
yeah we're not really consistent. I'm fine with this if you found it helpful